packages/common/package.json

```diff
@@ -41,12 +41,14 @@
   "type": "module",
   "types": "types/index.d.ts",
   "dependencies": {
+    "ajv": "^8.12.0",
     "axios": "1.1.3",
     "csv-parse": "^5.4.0",
     "csvtojson": "^2.0.10",
     "date-fns": "^2.25.0",
     "jsonpath-plus": "^4.0.0",
-    "lodash": "^4.17.19"
+    "lodash": "^4.17.19",
+    "undici": "^5.22.1"
```

Comment:
It's a bit annoying to carry undici and axios at the same time - but as axios will be removed before too long I think this OK to do.